webClient/src/app/editor/project-tree/project-tree.component.ts

```diff
@@ -229,6 +240,18 @@ export class ProjectTreeComponent {
     }
   }
 
+  onOpenInNewTab($event: any){
+    if ($event.data === 'File'){
+      const baseURI = `${window.location.origin}${window.location.pathname}`;
+      const newWindow = window.open(`${baseURI}?pluginId=org.zowe.editor:data:{"type":"openFile","name":"${encodeURIComponent($event.path)}","toggleTree":true}`, '_blank');
```

Comment:
org.zowe.editor shouldn't be hardcoded, should be able to find it with

@Inject(Angular2InjectionTokens.PLUGIN_DEFINITION) private pluginDefinition: ZLUX.ContainerPluginDefinition,